query-engine/driver-adapters/js/smoke-test-js/.envrc.example

```diff
@@ -1,3 +1,5 @@
+source_up
```

Comment:
I believe this comment will help when reading the code

```suggestion
# This will load the .envrc at the root of the `prisma-engines` repository before loading this one
source_up
```